app/utils/nonsipp/TaskListStatusUtils.scala

```diff
@@ -1003,4 +1003,114 @@ object TaskListStatusUtils {
       Updated
     }
   }
+
+  def userAnswersUnchangedAllSections(currentUA: UserAnswers, previousUA: UserAnswers): Boolean = {
+    (getBasicDetailsTaskListStatus(currentUA, previousUA) == Completed) &&
+    (getFinancialDetailsTaskListStatus(currentUA, previousUA) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.memberdetails.Paths.personalDetails,
+      Some("safeToHardDelete")
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.employercontributions.Paths.memberEmpContribution
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.memberpayments.Paths.membersPayments \ "unallocatedContribAmount"
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.membercontributions.Paths.memberDetails \ "totalMemberContribution"
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.receivetransfer.Paths.memberTransfersIn
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.membertransferout.Paths.memberTransfersOut
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.memberreceivedpcls.Paths.memberDetails \ "memberLumpSumReceived"
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.memberpensionpayments.Paths.memberDetails \ "pensionAmountReceived"
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.membersurrenderedbenefits.Paths.memberPensionSurrender
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.loansmadeoroutstanding.Paths.loans
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.moneyborrowed.Paths.borrowing
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.shares.Paths.shareTransactions,
+      Some("disposedSharesTransaction")
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
+      currentUA,
+      previousUA,
+      pages.nonsipp.sharesdisposal.Paths.disposedSharesTransaction
+    ) == Completed) &&
+    (getCompletedOrUpdatedTaskListStatus(
```

Comment:
If there are no land or property items, there are no disposals (menu item hidden). Would this logic still work in that scenario?

Reply:
FYI @BabuRupoll7623 could you please check if this is working as expected?

Reply:
Hidden disposal link working fine for (LoP, Bonds , 
shares and other assets)